hw/ip/pwrmgr/pwrmgr.core

```diff
@@ -32,6 +32,7 @@ filesets:
       # common waivers
       - lowrisc:lint:common
       - lowrisc:lint:comportable
+      - lowrisc:prim:flop_lint
```

Comment:
@msfschaffner 
what do you think?

Reply:
I wonder whether we should just place an RTL inline waiver into the abstract prim template that gets generated for now, since we have this issue with all generated prims. by doing that, I think it will in the future be easier to consolidate and solve the issue for real (e.g., by generating the waiver file via the primgen), rather than having to go through all designs that use the prims and remove this workaround... WDYT?

I think the waivers would look like this

inline, or previous line
```systemverilog
...code; //ri lint_check_waive <rule names>

//ri lint_check_waive <rule names>
...code;
```

block:
```systemverilog
//ri lint_check_off <rule names>
...code...
//ri lint_check_on
```

Reply:
that's fine with me..but...i thought we had style rules that prohibited inline things? Am I mistaken?

Reply:
o the other thing i wanted to add was that we will have more module level waivers in the future right? So wouldn't it make sense to absorb those into the top level as well?

Reply:
yes, we try to keep the waivers out of the code, since we have so many tools (and sometimes multiple tools that do the same thing, as in the case of linting), that this might start to clutter the code too much. however, I think in this particular case where the prims are generated, I think this here applies:
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#style-guide-exceptions

it would be temporary anyways, so I would suggest that we just add a `//TODO` comment and say that this inline waiver should eventually be moved into its own waiver file. WDYT?

Reply:
by module-level waivers... do you mean waivers for a particular comportable IP?

Reply:
yea that's right. If we think about how we might want to inherit those waivers elsewhere, wouldn't we need something like this?

Reply:
so I think the mechanism that you are referring to is already in place. if you look at some of the "more established" IPs, like HMAC, they already do exactly that:

https://github.com/lowRISC/opentitan/blob/31d877eb577dafba25dc8fd5f1c4b3ec19734156/hw/ip/hmac/hmac.core#L31-L38

I.e., they have a waiver file that is specific to that comportable IP, and that waiver file will get inherited by all dependencies that include this core if added in this way. 

This allows us to keep the waivers on the level they are needed. I.e., IP-level waivers should go into that waiver file, whereas top-level waivers should go into the top-level waiver file.

For the generated prims, I think we should do something similar - but since the lint violation occurs within generated code we would have to either place an in-line waiver into the primgen template:
https://github.com/lowRISC/opentitan/blob/31d877eb577dafba25dc8fd5f1c4b3ec19734156/hw/ip/prim/util/primgen/abstract_prim.sv.tpl#L1-L26
or generate a lint waiver alongside that gets linked in via the generated core file. I was suggesting the former solution since it is faster, but we may as well just have primgen generate this waiver file in order to be consistent.

Reply:
ahhh if that's the case.
Let's do it inline like you originally suggested then.  Because these waivers honestly "look" very scary.

If we have it in the design only for these modules, it will make more sense instead of looking at it standalone.
Let me remove this commit out of this PR first, and we can do that separately.

Reply:
yes sounds good!